app/src/pages/howTo/howto.js

```diff
@@ -1,5 +1,52 @@
 export default {
   name: 'HowTo',
+  data: () => ({
+    videos: []
+  }),
+  methods: {
+    showBox () {
+      console.log(this.filter)
+    },
+    hideVideos (idx) {
+      const vm = this
+      let noTouch = null
+      if (idx >= 0) {
+        noTouch = idx
+      }
+      vm.videos.forEach(function (v, i) {
+        if (i !== noTouch) {
+        // vm.videos[i].hide = true
+          const o = vm.videos[i]
+          o.hide = true
+          vm.$set(vm.videos, i, o)
+        }
+      })
+    },
+    displayVideo (idx, link) {
+      if (link) {
+        return window.open('https://youtu.be/o2FA1yM85M8', '_blank')
```

Comment:
It might be nice to pass the youtube link in as the value of `link` when you call `displayVideo` in `howto.html` (instead of passing a bool & hardcoding the link into the method), since we might add more how-to videos in the future that would re-use this method

Reply:
Good idea, fixed in 2df3b22